terraform/projects/infra-networking-dev/main.tf

```diff
@@ -0,0 +1,33 @@
+terraform {
+  required_version = "= 0.11.10"
+
+  backend "s3" {
+    bucket = "re-observe-dev"
+    key    = "infra-networking-dev.tfstate"
+  }
+}
+
+provider "aws" {
+  version = "~> 1.14.1"
+  region  = "${var.aws_region}"
+}
+
+variable "aws_region" {
+  type        = "string"
+  description = "The AWS region to use."
+  default     = "eu-west-1"
+}
+
+variable "stack_name" {
+  type        = "string"
+  description = "Unique name for this collection of resources"
+  default     = "ecs-monitoring"
+}
+
+module "infra-networking" {
+  source = "../../modules/infra-networking"
+
+  aws_region      = "eu-west-1"
+  dev_environment = true
+  stack_name      = "module-refactor"
+}
```

Comment:
we'll need to have `output` stanzas here for outputs to end up in the project statefile, so that they can be consumed by remote_state in other projects. 🍝

Reply:
Which outputs? [All of these](https://github.com/alphagov/prometheus-aws-configuration-beta/pull/196/files#diff-2a21f8981318c66bb6891f0f6b2edc40R105)?